js/parts/Chart.js

```diff
@@ -1242,24 +1361,34 @@ extend(Chart.prototype, /** @lends Highcharts.Chart.prototype */ {
      * Resize the chart to a given width and height. In order to set the width
      * only, the height argument may be skipped. To set the height only, pass
      * `undefined` for the width.
-     * @param  {Number|undefined|null} [width]
+     *
+     * @function Highcharts.Chart#setSize
+     *
+     * @param  {number|null} [width]
```

Comment:
We don't define `undefiend` anymore?

Reply:
In TypeScript every optional parameter implies support of `undefined`. So if you like to skip an optional parameter to get to an other optional parameter, you are always allowed to set the one to skip to `undefined`.

Reply:
Do you think, we should add to every optional parameter the `undefined` type, to make this clear?

Reply:
I'm just worried it might be confusing for users. Like "is undefined deprecated?"

Reply:
I see your point. I will add `undefined` to optionals, to make it clear for everyone.